docs/structure.md

```diff
@@ -2,8 +2,9 @@
 
 ## Hierachy
```

Comment:
I understand why you have this page as an "index" but could there be a way to automate this? I feel it would be very easy to miss methods/forget to update.

Reply:
Could be! The [macros plugin](https://mkdocs-macros-plugin.readthedocs.io/en/latest/) lets you define arbitrary Python functions that you can invoke in markdown like this: `{{ mymacro(...) }}`    Already tested it, works super well. Can even produce plots etc.    Not 100% sure if it would work here, but think so. Would need to get a list of the functions and return it formatted like this.